proto/src/determined/api/v1/api.proto

```diff
@@ -21,33 +22,59 @@ import "determined/api/v1/shell.proto";
 import "determined/api/v1/user.proto";
 
 option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
-    info: {
-        title: "Determined API (Beta)",
-        description: "Determined helps deep learning teams train models more quickly, easily share GPU resources, and effectively collaborate. Determined allows deep learning engineers to focus on building and training models at scale, without needing to worry about DevOps or writing custom code for common tasks like fault tolerance or experiment tracking.\n\nYou can think of Determined as a platform that bridges the gap between tools like TensorFlow and PyTorch --- which work great for a single researcher with a single GPU --- to the challenges that arise when doing deep learning at scale, as teams, clusters, and data sets all increase in size.",
-        version: "0.1",
-        contact: {
-            name: "Determined AI",
-            url: "https://determined.ai/",
-            email: "community@determined.ai",
-        }
-        license: {
-            name: "Apache 2.0",
-            url: "http://www.apache.org/licenses/LICENSE-2.0.html"
-        }
-    }
-    schemes: [HTTP, HTTPS],
-    external_docs: {
-        description: "Determined AI Documentation",
-        url: "/docs",
-    }
+  info : {
+    title : "Determined API (Beta)",
+    description : "Determined helps deep learning teams train models more "
+                  "quickly, easily share GPU resources, and effectively "
+                  "collaborate. Determined allows deep learning engineers to "
+                  "focus on building and training models at scale, without "
+                  "needing to worry about DevOps or writing custom code for "
+                  "common tasks like fault tolerance or experiment "
+                  "tracking.\n\nYou can think of Determined as a platform that "
+                  "bridges the gap between tools like TensorFlow and PyTorch "
+                  "--- which work great for a single researcher with a single "
+                  "GPU --- to the challenges that arise when doing deep "
+                  "learning at scale, as teams, clusters, and data sets all "
+                  "increase in size.",
+    version : "0.1",
+    contact : {
+      name : "Determined AI",
+      url : "https://determined.ai/",
+      email : "community@determined.ai",
+    }
+    license : {
+      name : "Apache 2.0",
+      url : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  }
+  schemes : [ HTTP, HTTPS ],
+  external_docs : {
+    description : "Determined AI Documentation",
+    url : "/docs",
+  },
```

Comment:
guessing commas are optional? just seeing inconsistencies in the protobuf. For example, `external_docs` has a comma at the end of the dictionary, but not for `license`, `contact` nor `info` (and other places too)

Reply:
yeah optional I couldn't find a reference to a formatting guide that mentions that but this is how grpc_gateway people do it. https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/a_bit_of_everything.proto